Misc/NEWS.d/next/Library/2021-07-20-00-11-47.bpo-44682.3m2qVV.rst

```diff
@@ -0,0 +1,2 @@
+In :mod:`pdb`, *commands* directive was fixed to disallow setting commands
+for a non-existing breakpoint and to display "out of range" error.
```

Comment:
It's not always "out of range" - if you look at Bdb.get_bpbynumber there are a number of other possible error messages.